openmetadata-ui/src/main/resources/ui/src/styles/variables.less

```diff
@@ -158,6 +158,7 @@
 @switch-sm-min-width: 23px;
 
 // Sizes to use commonly for padding, margin, etc.
+@size-0: 2px;
```

Comment:
name should be different here, "{anything}-0" is semantically used in other places to define 0 value,
Also we are using sizing in multiples of 4, if you need 2px in some case you should be defining it in your component.